extensions/markdown-language-features/notebook/index.ts

```diff
@@ -24,5 +24,10 @@ type extendMarkdownItFnType = (
 	notebook.onDidCreateMarkdown(({ element, content }: any) => {
 		const rendered = markdownIt.render(content);
 		element.innerHTML = rendered;
+
+		// Insert styles into markdown preview shadow dom so that they are applied
+		for (const markdownStyleNode of document.getElementsByClassName('markdown-style')) {
```

Comment:
In the future, we should look into pulling in contributed styles through some explicit contribution point instead of this implicit convention